src/app/index.tsx

```diff
@@ -32,15 +33,15 @@ export const actions = {
   notifications: notifications.actions,
 };
 
-export const routes = Object.values({
+export const routes = assertNonNullableArray(Object.values({
   ...(
     process.env.REACT_APP_ENV !== 'production'
       ? developer.routes
       : /* istanbul ignore next */ {}
   ),
   ...content.routes,
   ...errors.routes,
-});
+}), 'some of the required routes are not exported');
```

Comment:
how is it possible that these values would be null?

Reply:
it more about ts not being able to recognize that we are exporting `routes` from each of these routes so when we want to use it it throws:
```ts
const routes: (navigation.types.Route<{}, {}> | navigation.types.Route<content.types.Params, State> | navigation.types.Route<Params, {}> | undefined)[]
```
```
Argument of type '(Route<{}, {}> | Route<Params, State> | Route<Params, {}> | undefined)[]' is not assignable to parameter of type 'AnyRoute[]'.
  Type 'Route<{}, {}> | Route<Params, State> | Route<Params, {}> | undefined' is not assignable to type 'AnyRoute'.
    Type 'undefined' is not assignable to type 'AnyRoute'.ts(2345)
```
I've used `assertNonNullableArray` name because I've also used `NonNullable` for `export type AnyRoute = NonNullable<typeof routes[number]>;`

Reply:
what is actually causing that undefined to be in the type? is it one of the modules in particular that has it?

Reply:
is it because of the ternary? that is very odd

it would be safe to cast the ternary result to `typeof developer.routes` if that helps